frontend/src/app/globals.css

```
@@ -1871,3 +1871,234 @@ input[type="reset"]:disabled {
+.landing-dc ::-webkit-scrollbar-thumb { background: rgba(0,0,0,0.15); border-radius: 3px; }
+.landing-dc .gal-rail                     { scrollbar-width: none; }
+.landing-dc .gal-rail::-webkit-scrollbar  { display: none; }
+
+/* ── Intro lockup ──────────────────────────────────────────────────
+   A hand-tuned causal sequence, 2.4s, played once with `both`. The stem
+   passes each joint before that part grows; the leaves then settle under
+   their own weight. The per-keyframe animation-timing-function
+   declarations ARE the effect — collapsing them onto one shared curve
+   destroys it. Do not "simplify" these.
+   ────────────────────────────────────────────────────────────────── */
+@keyframes s1Stem   { 0%,15% { stroke-dashoffset:27; } 34%,100% { stroke-dashoffset:0; } }
```

Stylelint (17.14.0)

[error] Expected keyframe name "s1Stem" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)